ContractExamples/scripts/alert-deploy.sh

```diff
@@ -25,4 +29,7 @@ set -x
 ./scripts/build.sh
 
 stellar contract deploy --wasm target/wasm32-unknown-unknown/release/alert.wasm \
-      --source $ACCOUNT --network $NET
\ No newline at end of file
+      --source $ACCOUNT --network $NET | tee >.alert.id
+
+CONTRACT_HASH=`head -n 1 .alert.id`
```

Comment:
I think this is the (contract) address, not a (WASM) hash?

Reply:
yes, this is confusing. The tutorials seem to call it a contract id. Let's rename to `CONTRACT_ADDR`.